styles.css

```diff
@@ -78,11 +94,264 @@
+  color: 344563;
+}
+
+#back-end {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  color: #7a969a;
+}
+
+#project-detail {
+  font-family: poppins, sans-serif;
+  font-weight: 600px;
+  font-size: 15px;
+  font-style: normal;
+  color: #344563;
+}
+
+.lang-list {
+  font-family: 'Poppins', sans-serif;
+  display: flex;
+  padding: 0;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  align-items: center;
+  color: #6070ff;
+  gap: 10px;
+  list-style-type: none;
+}
+
+#lang {
+  padding: 4px 12px;
+  border-radius: 8px;
+  color: #6070ff;
+  background: #ebebff;
+}
+
+button {
+  width: 150px;
+  padding: 12px;
+  background: #fff;
+  border: 2px solid #396df2;
+  border-radius: 10px;
+  color: #396df2;
+  font-size: 17px;
+  margin: 40px 16px;
+  display: flex;
+  align-items: center;
+  gap: 10px;
+  line-height: 24px;
+  flex-direction: column;
+  font-weight: 700;
+}
+
+button:hover {
+  background-color: #6070ff;
+  color: #fff;
+}
+
+button:active {
+  background-color: #2230d2;
+  color: #fff;
+}
+
+.about {
+  display: flex;
+  flex-direction: column;
+}
+
+#about-me {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  margin-left: 24px;
+}
+
+#about-myself {
+  margin-left: 24px;
+  color: #344563;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+#get-resume {
+  width: 158px;
+  font-size: 17px;
+}
+
+.dropdown-arrow {
+  width: 0;
+  height: 0;
+  border-left: 5px solid transparent;
+  border-right: 5px solid transparent;
+  border-bottom: 5px solid black;
+}
+
+.prog-languages {
+  list-style-type: none;
+  display: flex;
+  gap: 120px;
+}
+
+#languages-header {
+  font-family: poppins, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  color: #000;
+}
+
+.languages-list {
+  list-style-type: none;
+}
+
+.item {
+  width: 303px;
+  display: flex;
+  align-items: center;
+  gap: 15px;
+  background: #f7f7f9;
+  border-radius: 8px;
+  font-family: poppins, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  color: #253858;
+  height: 66.72px;
+  margin-bottom: 12px;
+}
+
+.frameworks {
+  list-style-type: none;
+}
+
+#frameworks-1 {
+  display: flex;
+  gap: 170px;
+  align-items: center;
+}
+
+#frameworks-2 {
+  display: flex;
+  align-items: center;
+  gap: 212px;
+}
+
+#contact-container {
+  background: #6070ff;
+  border-top-left-radius: 55px;
+  width: 375px;
+  background-image: url(/media/contact\ form\ shapes-mobile.svg);
+  background-position: 100% 3%;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 0;
+
+  /* margin:-8px; */
+}
```

Comment:
- [ ] Your form is overflowing on the mobile version. You need to give it a [box-sizing](https://www.w3schools.com/css/css3_box-sizing.asp) property. to fix this flaw.

_**figma**_
![box-1](https://user-images.githubusercontent.com/99656687/213687144-90fa608d-f4c1-4fdd-ae29-09539af0b922.png)

_**your design**_
![box-2](https://user-images.githubusercontent.com/99656687/213687310-d08d0b1e-ba05-49a1-990a-43c5d2118b0a.png)